tools/cereal-scaffold/main.go

```diff
@@ -199,20 +233,211 @@ func runResetDB(_ *cobra.Command, args []string) error {
 	return nil
 }
 
+func getBackend() backend.Driver {
+	if opts.Endpoint != "" {
+		conn, err := grpc.Dial(opts.Endpoint, grpc.WithInsecure(), grpc.WithMaxMsgSize(64*1024*1024))
+		if err != nil {
+			panic(err)
+		}
+		grpcBackend := grpccereal.NewGrpcBackendFromConn("test", conn)
+		return grpcBackend
+	}
+	return postgres.NewPostgresBackend(defaultConnURIForDatabase())
+}
+
+var simpleWorkflowStartTime *time.Time
```

Comment:
This is a bit of a hack.  It could be removed if we passed more data to our OnWorkflowComplete callback.